src/main/java/io/vertx/core/net/impl/SSLHelper.java

```diff
@@ -133,6 +130,9 @@ public static SSLEngineOptions resolveEngineOptions(TCPSSLOptions options) {
   private boolean trustAll;
   private ArrayList<String> crlPaths;
   private ArrayList<Buffer> crlValues;
+  // TODO - HTTP Server has own instance of SSL Helper. How do we invalidate/reload CRLs for
+  // TODO - all instances of HTTP Server. For now making it static. Lets see what Julien think.
+  private static List<CRL> crls;
```

Comment:
Right now there is open question on how we invalidate in-memory CRL in HttpServer/HttpClient.

HttpServer instances (and SslHelper as well) are created per-thread, so user will have to ensure that they store HttpServer instance somewhere and call the reload method on each one of them in loop. I need recommendation from Vertx team on this. Right now I have kept it static for my testing purpose but it can't be kept static because SSL helper is shared among HttpClient & HttpServer.